src/kerberos/mod.rs

```diff
@@ -653,7 +653,7 @@ impl<'a> Kerberos {
         Ok(())
     }
 
-    pub async fn initialize_security_context_impl(
+    pub(crate) async fn initialize_security_context_impl(
```

Comment:
stay consistent with NTLM, pku2u, make it easier to call with only
```
kerberos.initialize_security_context_impl(&mut builder)
```
as shown in example kerberos.rs